src/System.IO.FileSystem.AccessControl/tests/DirectoryObjectSecurityTests.cs

```diff
@@ -19,6 +19,15 @@ public class DirectoryObjectSecurityTests
         private const int WriteAccessMask = 0x02;
         private const int ReadAttributeAccessMask = 0x80;
 
+        [Fact]
+        public void SetCustomDescriptor_Success()
+        {
+            // https://github.com/dotnet/corefx/issues/34151
```

Comment:
What is the comment supposed to be saying to me?

Reply:
That there is a related issue for this test. I can spell it out.